QtCollider/widgets/image_painter.h

```diff
@@ -144,7 +144,6 @@ struct ImagePainter
         bool tileVertically = verticalMode == TileVertically;
         bool tileHorizontally = horizontalMode == TileHorizontally;
 
-        qreal x_origin = rect.left();
```

Comment:
a call to `moveLeft` updates the value returned by `rect.left()`. After `i` calls, its value will be `i * rect.width() + x_origin`.